soccer_vision_3d_msgs/msg/BallArray.msg

```diff
@@ -0,0 +1,5 @@
+# An array of balls with a header for global reference.
+# This msg type is used to publish all balls detected in an image.
```

Comment:
`detected in an image` I guess it is still the case, but it sounds a little bit odd...

Reply:
I agree, I'm going to remove ``in an image``